scene/main/viewport.cpp

```diff
@@ -67,7 +67,7 @@ void ViewportTexture::setup_local_to_scene() {
 	}
 
 	Node *vpn = local_scene->get_node(path);
-	ERR_FAIL_COND_MSG(!vpn, "ViewportTexture: Path to node is invalid.");
+	ERR_FAIL_COND_MSG(!vpn, "ViewportTexture: Path to node is invalid. If the path seems valid, move the node using the texture after the viewport.");
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(!vpn, "ViewportTexture: Path to node is invalid. If the path seems valid, move the node using the ViewportTexture so that it is placed after the Viewport in the scene tree.");
```